cli/test/lib/tasks/state_spec.js

```diff
@@ -258,6 +259,16 @@ describe('lib/tasks/state', function () {
 
       expect(ret).to.eql(path.resolve('local-cache/folder'))
     })
+
+    it('resolves ~ with user home folder', () => {
+      process.env.CYPRESS_CACHE_FOLDER = '~/.cache/Cypress'
+
+      const ret = state.getCacheDir()
+
+      debug('cache dir is "%s"', ret)
+      expect(path.isAbsolute(ret), ret).to.be.true
+      expect(ret, '~ has been resolved').to.not.contain('~')
```

Comment:
wish we used mockfs in this spec, so we could just assert on the actual value, but this works too

Reply:
I don't think we could - `untildify` is a regex that uses `os. homedir()` - actually, let me do this